### ISSUE TITLE
useBreadcrumbItems() Maximum call stack size exceeded

### ISSUE THREAD

**Issue by marcelxpfeifer** [harlan-zw/nuxt-seo-kit#148]
### Describe the bug

After installing nuxt i18n the UBreadcrumb in combination with useBreadcrumbItems throws an error.

```
[nuxt] [request error] [unhandled] [500] Maximum call stack size exceeded
  at Module.get [as stringifyParsedURL] (./node_modules/.pnpm/ufo@1.3.2/node_modules/ufo/dist/index.mjs:568:110)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:86:44)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
  at stepNode (./node_modules/.pnpm/@nuxtseo+module@2.0.0-beta.51_@nuxt+devtools@1.0.6_@unhead+shared@1.8.9_@vue+compiler-core@3._3ib7dlp5yt4waxlpuqksfyszsq/node_modules/@nuxtseo/module/dist/runtime/composables/useBreadcrumbItems.mjs:93:7)
```

Haven't had the time to create a minimal reproduction, but I will try to add it in a few days.

### System / Nuxt Info

```Shell
System:
    OS: macOS 14.2.1
    CPU: arm64 Apple
  Binaries:
    Node: 18.7.0 - ~/.nvm/versions/node/v18.7.0/bin/node
    npm: 8.15.0 - ~/.nvm/versions/node/v18.7.0/bin/npm
    pnpm: 8.13.1 - ~/.nvm/versions/node/v18.7.0/bin/pnpm

i18n: v1.0.0
seo kit: v2.0.0-beta.51
ui: v2.11.1
```

**Comment by harlan-zw** [comment]
Hi, will investigate this one further. Do you have the path that triggered this though?

**Comment by marcelxpfeifer** [comment]
It was two levels deep so 
/ 
/x 
/x/y 

All had names and icons set

**Comment by harlan-zw** [comment]
Thanks, should be fixed in v2.0.0-beta.52.
